src/demo/css/style.css

```diff
@@ -0,0 +1,1983 @@
+  }
+  .row .col-xl-6 {
+    max-width: 50%;
+    flex: 0 0 50%;
+  }
+  .row .col-xl-7 {
+    max-width: 58.33%;
+    flex: 0 0 58.33%;
+  }
+  .row .col-xl-8 {
+    max-width: 66.66%;
+    flex: 0 0 66.66%;
+  }
+  .row .col-xl-9 {
+    max-width: 75%;
+    flex: 0 0 75%;
+  }
+  .row .col-xl-10 {
+    max-width: 83.33%;
+    flex: 0 0 83.33%;
+  }
+  .row .col-xl-11 {
+    max-width: 91.66%;
+    flex: 0 0 91.66%;
+  }
+  .row .col-xl-12 {
+    max-width: 100%;
+    flex: 0 0 100%;
+  }
+}
+
+.dark-theme {
+  --gray-900: #ffffff;
+  --gray-800: #f7f7f8;
+  --gray-700: #e7e9eb;
+  --gray-600: #c0c3c9;
+  --gray-500: #b0b4bb;
+  --gray-400: #52555d;
+  --gray-300: #292c34;
+  --gray-200: #1b1d22;
+  --gray-100: #0a0b0d;
+  --primary-900: #d2bdf5;
+  --primary-800: #c1a5f2;
+  --primary-700: #af89ee;
+  --primary-600: #996aea;
+  --primary-500: #8247e5;
+  --primary-400: #693ab9;
+  --primary-300: #532e92;
+  --primary-200: #402370;
+  --primary-100: #2f1a52;
+  --secondary-900: #bfe2ea;
+  --secondary-800: #a9d8e2;
+  --secondary-700: #8eccd9;
+  --secondary-600: #71becf;
+  --secondary-500: #4fafc4;
+  --secondary-400: #408d9e;
+  --secondary-300: #32707e;
+  --secondary-200: #265560;
+  --secondary-100: #1c3f47;
+  --success-900: #beebe5;
+  --success-800: #a6e3dc;
+  --success-700: #8bdad1;
+  --success-600: #6cd1c4;
+  --success-500: #4ac6b7;
+  --success-400: #3ca095;
+  --success-300: #2f7e75;
+  --success-200: #24615a;
+  --success-100: #1a4742;
+  --warning-900: #fdddc1;
+  --warning-800: #fcd1aa;
+  --warning-700: #fbc390;
+  --warning-600: #fbb272;
+  --warning-500: #f9a151;
+  --warning-400: #ca8342;
+  --warning-300: #9f6734;
+  --warning-200: #7a4f28;
+  --warning-100: #593a1d;
+  --error-900: #fcbebe;
+  --error-800: #fba6a6;
+  --error-700: #f98b8b;
+  --error-600: #f86c6c;
+  --error-500: #f64a4a;
+  --error-400: #c73c3c;
+  --error-300: #9e2f2f;
+  --error-200: #782424;
+  --error-100: #591a1a;
+}
+
+.red-theme {
+  --gray-900: #ffffff;
+  --gray-800: #f7f7f8;
+  --gray-700: #e7e9eb;
+  --gray-600: #c0c3c9;
+  --gray-500: #b0b4bb;
+  --gray-400: #52555d;
+  --gray-300: #292c34;
+  --gray-200: #1b1d22;
+  --gray-100: #0a0b0d;
+  --primary-900: #fcbebe;
+  --primary-800: #fba6a6;
+  --primary-700: #f98b8b;
+  --primary-600: #f86c6c;
+  --primary-500: #f64a4a;
+  --primary-400: #c73c3c;
+  --primary-300: #9e2f2f;
+  --primary-200: #782424;
+  --primary-100: #591a1a;
+}
+
+body {
+  font-family: "Manrope", sans-serif;
+  background-color: var(--gray-300);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  color: var(--gray-900);
+}
+
+.section {
+  width: 90%;
+  margin: 50px auto;
+  padding: 30px 40px;
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.03), 0px 12px 8px rgba(0, 0, 0, 0.02);
+}
+.section.center {
+  display: flex;
+  justify-content: center;
+}
+
+.section-container {
+  display: inline-flex;
+  flex-direction: column;
+  margin: 10px;
+  max-width: 570px;
+}
+.section-container.hr {
+  max-width: none;
+}
+
+.group {
```

Comment:
not sure about this, let us discuss this

Reply:
This is not part of the design system. It is used for the demo purpose.